tests/filecheck/dialects/arith/arith_ops.mlir

```diff
@@ -10,142 +10,215 @@
 
   %mulf = "arith.mulf"(%lhsf32, %rhsf32) : (f32, f32) -> f32
+  %cmulf = arith.mulf %lhsf32, %rhsf32 : f32
   %mulf_vector = "arith.mulf"(%lhsvec, %rhsvec) : (vector<4xf32>, vector<4xf32>) -> vector<4xf32>
+  %cmulf_vector = arith.mulf %lhsvec, %rhsvec : vector<4xf32>
 
-  // CHECK-NEXT: %mulf = "arith.mulf"(%lhsf32, %rhsf32) : (f32, f32) -> f32
-  // CHECK-NEXT: %mulf_vector = "arith.mulf"(%lhsvec, %rhsvec) : (vector<4xf32>, vector<4xf32>) -> vector<4xf32>
+  // CHECK-NEXT: %{{c?}}mulf = "arith.mulf"(%lhsf32, %rhsf32) : (f32, f32) -> f32
+  // CHECK-NEXT: %{{c?}}mulf = "arith.mulf"(%lhsf32, %rhsf32) : (f32, f32) -> f32
+  // CHECK-NEXT: %{{c?}}mulf_vector = "arith.mulf"(%lhsvec, %rhsvec) : (vector<4xf32>, vector<4xf32>) -> vector<4xf32>
+  // CHECK-NEXT: %{{c?}}mulf_vector = "arith.mulf"(%lhsvec, %rhsvec) : (vector<4xf32>, vector<4xf32>) -> vector<4xf32>
 
   %divf = "arith.divf"(%lhsf32, %rhsf32) : (f32, f32) -> f32
+  %cdivf = arith.divf %lhsf32, %rhsf32 : f32
   %divf_vector = "arith.divf"(%lhsvec, %rhsvec) : (vector<4xf32>, vector<4xf32>) -> vector<4xf32>
+  %cdivf_vector = arith.divf %lhsvec, %rhsvec : vector<4xf32>
 
-  // CHECK-NEXT: %divf = "arith.divf"(%lhsf32, %rhsf32) : (f32, f32) -> f32
-  // CHECK-NEXT: %divf_vector = "arith.divf"(%lhsvec, %rhsvec) : (vector<4xf32>, vector<4xf32>) -> vector<4xf32>
+  // CHECK-NEXT: %{{c?}}divf = "arith.divf"(%lhsf32, %rhsf32) : (f32, f32) -> f32
+  // CHECK-NEXT: %{{c?}}divf = "arith.divf"(%lhsf32, %rhsf32) : (f32, f32) -> f32
+  // CHECK-NEXT: %{{c?}}divf_vector = "arith.divf"(%lhsvec, %rhsvec) : (vector<4xf32>, vector<4xf32>) -> vector<4xf32>
+  // CHECK-NEXT: %{{c?}}divf_vector = "arith.divf"(%lhsvec, %rhsvec) : (vector<4xf32>, vector<4xf32>) -> vector<4xf32>
 
   %negf = "arith.negf"(%lhsf32) : (f32) -> f32
 
-  // CHECK-NEXT: %negf = "arith.negf"(%lhsf32) : (f32) -> f32
+  // CHECK: %negf = "arith.negf"(%lhsf32) : (f32) -> f32
 
   %extf = "arith.extf"(%lhsf32) : (f32) -> f64
 
-  // CHECK-NEXT: %extf = "arith.extf"(%lhsf32) : (f32) -> f64
+  // CHECK: %extf = "arith.extf"(%lhsf32) : (f32) -> f64
 
   %truncf = "arith.truncf"(%lhsf64) : (f64) -> f32
 
-  // CHECK-NEXT: %truncf = "arith.truncf"(%lhsf64) : (f64) -> f32
+  // CHECK: %truncf = "arith.truncf"(%lhsf64) : (f64) -> f32
 
   %cmpf = "arith.cmpf"(%lhsf32, %rhsf32) {"predicate" = 2 : i64} : (f32, f32) -> i1
 
-  // CHECK-NEXT: %cmpf = "arith.cmpf"(%lhsf32, %rhsf32) {"predicate" = 2 : i64} : (f32, f32) -> i1
+  // CHECK: %cmpf = "arith.cmpf"(%lhsf32, %rhsf32) {"predicate" = 2 : i64} : (f32, f32) -> i1
 
   %selecti = "arith.select"(%lhsi1, %lhsi32, %rhsi32) : (i1, i32, i32) -> i32
   %selectf = "arith.select"(%lhsi1, %lhsf32, %rhsf32) : (i1, f32, f32) -> f32
 
-  // CHECK-NEXT: %selecti = "arith.select"(%lhsi1, %lhsi32, %rhsi32) : (i1, i32, i32) -> i32
-  // CHECK-NEXT: %selectf = "arith.select"(%lhsi1, %lhsf32, %rhsf32) : (i1, f32, f32) -> f32
```

Comment:
Why are we dropping the `CHECK-NEXT` here?

Reply:
For no good reason, good catch :)
I expected some feedback and rework to happen on this file eitherway it didn't thoroughly checked everything.
I'll rework the file now and pay attention to this!